src/app.rs

```diff
@@ -2251,6 +2277,59 @@ impl TaskwarriorTui {
     }
   }
 
+  pub async fn task_edit_markdown(&mut self) -> Result<(), String> {
+    if self.tasks.is_empty() {
+      return Ok(());
+    }
+
+    self.pause_tui().await.unwrap();
+
+    let selected = self.current_selection;
+    let task_id = self.tasks[selected].id().unwrap_or_default();
+    let task_uuid = *self.tasks[selected].uuid();
+
+    let data_dir = shellexpand::tilde(&self.config.data_location).into_owned();
+
+    let r = std::process::Command::new("nvim")
```

Comment:
We shouldn't hard code this to `nvim`, right? People may want to use any editor.

Reply:
yes,yeah